sdks/python/apache_beam/io/tfrecordio_test.py

```diff
@@ -405,6 +405,9 @@ def test_process_auto(self):
         assert_that(result, equal_to(['foo', 'bar']))
 
 
+@unittest.skipIf(sys.version_info[0] == 3,
```

Comment:
There was a change recently in TFRecordIO: https://github.com/apache/beam/commit/5bb33a6666a131ebd6976fdf5a31b3d6e7689f53#diff-6f2f08b1aad2a8511b307fb8736da843 - is this test still failing?

Reply:
This test hangs when executed on the master branch both in my local environment as well as in an environment I set up on a VM. @RobbeSneyders verified this and this is also the case in his environments.

The reason all IO changes pass the jenkins checks is because in tox.ini none of the IO tests are being added and are therefore are not checked. They are of course presumably being tested locally, so it rarely results in an issue, but if the local check is done before a rebase this could conflict with another change imo, although in an indirect manner because otherwise there would be a merge conflict. Do you think we should add the altered io tests to the tox file from now on and go back to skipping remaining errors in order to pass the jenkins environment?

Reply:
The more tests we run on Jenkins in Python 3, the more Py3 regressions we can capture.
If 1 out of 10 tests in the package is passing, it may be easier to enlist one test in tox.ini, than add all tests in the package and skip 9 that are not passing.  If 9 out of 10 tests are passing, we can file a JIRA for a remaining tests, if it's failing due to a self-contained failure and skip it, and add the package. 
If we think that the whole package will be fixed in ~a week, it may not be worth the churn to add skipping instructions and add tests one-by-one.